spec/Section 3 -- Type System.md

```diff
@@ -757,20 +757,22 @@ Union types have the potential to be invalid if incorrectly defined.
 
 #### Union type syntax
 
-Union types are a list of types joined with a vertical bar `|`
+Union types are defined by delimiting one or more types with a single vertical bar character `|`:
 
 ```graphql
 union SearchResult = Photo | Person
 ```
 
-You may also use a leading vertical bar
+You may also use a leading vertical bar.
```

Comment:
Change this to `You may also use a leading vertical bar:` to be consistent with the opening clause.